styles/_app.scss

```diff
@@ -128,6 +128,13 @@
 // hide filter categories on mobile
 @media only screen and (max-width: 750px) {
   .categories-filter {
+    width: 100%;
```

Comment:
Instead of specifying the width in the stylesheet here, we can just use `col-xs-12 col-md-3` in the template.

Reply:
No, it does not work, you can try. It's fixed with: `max-width: 750px`. `col-*` don't work anymore.

Reply:
Thanks for bringing that to my attention, I missed the `max-width: 750px`. We already have some breakpoints ([sample usage](https://github.com/electron/electronjs.org/blob/master/styles/_base.scss#L7)) defined with basecoat, so we shouldn't add another one at 750px. That way we can keep using `col-*` for consistency.

Reply:
@zeke what do you think about refactoring the design of apps page to use `col-*`?